doc/source/examples/trustscore_iris.ipynb

```diff
@@ -2,14 +2,22 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
```

Comment:
A lot of cells seem to have this extra `pycharm` metadata added, could you please remove?

If this happens a lot when viewing notebooks in `pycharm` we can go ahead and not worry about this as it shouldn't impact functionality in any way. I note that @ascillitoe had a similar thing crop up.

Reply:
We might also be able to automate a cleaning step as a precommit hook with `nbstripout` or similar (there are other annoying bits of metadata that can sneak in such as `kernel`... Not sure its worth the effort compared to just checking the PR diff manually and removing though...

Reply:
Ok, I've submitted #644 for `nbstripout` as a pre-commit hook.